README.md

```diff
@@ -4,8 +4,7 @@ The Artwork gem provides simple server-side responsive images support for Rails
 which is similar in concept to the `<picture>` tag spec but requires no
 JavaScript, doesn't make extra requests and works in all browsers.
 
-Currently it supports only Paperclip attachments, but that can be changed
-failry easily.
+You could use it with Paperclip attachments or implement couple of method in plain Ruby.
```

Comment:
"You could use it either with Paperclip attachments or implement a couple of methods yourself in plain Ruby."